### PR TITLE
New(design): Exercises notes

### PR DESCRIPTION
**Pull request by camjc**
Some notes on how we might globally approach adding new exercises, some concerns including:
- We need to think about how to teach a user how to do the exercise without getting in their way if they're coming back for the nth time.
- We need to allow the user to configure the exercise without overwhelming them or being present when they're just trying to complete their daily exercise.

I'd appreciate your notes @xarvh

### CODE REVIEW

**Comment by xarvh** [comment]
The idea I was considering was to use sensible defaults and keep all settings in the browser local storage.
The main page will link to a task's instructions, which in turn will have a direct link to the task proper.
So a user can a) bookmark the task page directly or b) get there from the main page in two clicks.

The point where I am less clear about is how to provide the user with config options without getting in the way.
Things that are configurable are: language, duration, initial ISI.
The last two can stay a bit away, the first one should be a bit more visible, and possibly default to the browser's language.

The download buttons could be displayed after the first time Stop is pressed,

**Comment by camjc** [comment]
@xarvh Have been thinking about some of this, revised notes are up as a second commit. Happy to squash or make further changes, let me know.

File: design/Exercises.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Needs
+- I need encouragement so that I keep using the app past the initial frustration.
+- I need to be able to use the app for extended periods without hurting my eyes.
+- I need the exercises to work on any device and be accessibility friendly.
+- I need to be able to complete the tasks effectively (audio in my own language and other exercise specific concerns).
+- I need to be able to use it without an internet connection.
+
+
+## Workflow
+- Present the exercise, with a button to start, skippable (present if small, first part present if needs to be longer) instructions, and optional settings.
```
Comment (xarvh) [inline design/Exercises.md]: Can you clarify this? I think I get the general meaning, but I get lost in the specifics.

File: design/Exercises.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- The instructions teach the user how to complete the exercise, and explain why it helps.
+- Settings all have the most common case as default, changes are persistent without compromising privacy.
+- Let the user perform the exercise without getting in their way (settings, instructions etc all disappear once exercise has started).
+- User downloads their data (download before feedback so it feels necessary to progress).
```
Comment (xarvh) [inline design/Exercises.md]: Do we need to encourage the user to download their data at all?

**Comment by xarvh** [comment]
Added a couple of comments.
Feel free to squash or not, I don't mind either.
I think I'll start moving the codebase to React+Babel, so that we can handle more moving parts and have better portability.

**Comment by camjc** [comment]
@xarvh Merge if LGTY.

**Pull request merged**